pkg/daemon/cleanup/cleanup.go

```diff
@@ -92,6 +93,8 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, preserveNode bool) error
 
 // removeServices uses the given manager to remove all the given Windows services from this instance.
 func removeServices(svcMgr manager.Manager, services []servicescm.Service) error {
+	wicdSvc := &servicescm.Service{Name: windows.WicdServiceName}
```

Comment:
So WICD is removing itself? If yes, that would be an anti-pattern. WMCO should be the one removing WICD given it is the entity that installed it.

Reply:
I agree with this. I had it that way before, I think I just made this change as it was cleaner. Thanks for pointing out, will make the change.